components/molecules/AuthSection/auth-section.tsx

```diff
@@ -149,7 +149,10 @@ const AuthSection: React.FC = ({}) => {
             <Button
               variant="primary"
               onClick={async () =>
-                await signIn({ provider: "github", options: { redirectTo: `${host}${currentPath}` } })
+                await signIn({
+                  provider: "github",
+                  options: { redirectTo: `${host}${currentPath}`, scopes: "read:org" },
```

Comment:
Note: I had to sign out and sign back in on an incognito browser to get the higher scope auth flow to kick off.

Reply:
I wonder if we need to indicate that they may need to log out, or I wonder if there's a way for us to get the higher auth scope to catch?

Reply:
It seems the whole auth flow needs to be re-triggered. I got this email once I did:

```
Hey jpmcb!

Previously, a third-party OAuth application (auth-beta-opensauced) had user:email scope. It was recently granted an additional scope (read:org). Visit ... for more information.

To see this and other security events for your account, visit https://github.com/settings/security-log

If you run into problems, please contact support by visiting https://github.com/contact

Thanks,
The GitHub Team
```

which makes me think that it'll fail until the higher auth is granted explicitly by the user.

Reply:
Yes, that's true about the new auth flow. We could do an API check when they click the first button and redirect them if necessary, but thats adds a bit of friction.

Reply:
This was enough friction from getting the Facebook OSPO from using the original open-sauced/open-sauced.

This is partly why installing the GitHub is the path we should introduce for progressively getting access to more read only data. 

I believe the on wording screen has a note about reading public data only. If not, it should.